Formula/gcc.rb

```diff
@@ -70,7 +70,7 @@ def install
     #  - Ada, which requires a pre-existing GCC Ada compiler to bootstrap
     #  - Go, currently not supported on macOS
     #  - BRIG
-    languages = %w[c c++ objc obj-c++ fortran]
+    languages = %w[c c++ objc obj-c++ d fortran]
```

Comment:
```suggestion
    languages = %w[c c++ objc obj-c++ fortran]
    languages << "d" if Hardware::CPU.intel?
```
You'll have to do the same with the `args` array, `bin.install_symlink` and the test.

Reply:
thank you!